components/governance/org.wso2.carbon.governance.generic.ui/src/main/resources/web/generic/configure.jsp

```diff
@@ -37,133 +40,147 @@
-    } catch (Exception e){
+    } catch (Exception e) {
         response.setStatus(500);
         CarbonUIMessage uiMsg = new CarbonUIMessage(CarbonUIMessage.ERROR, e.getMessage(), e);
         session.setAttribute(CarbonUIMessage.ID, uiMsg);
 %>
-        <jsp:include page="../admin/error.jsp?<%=e.getMessage()%>"/>
+<jsp:include page="../admin/error.jsp?<%=e.getMessage()%>"/>
 <%
         return;
     }
 %>
 <fmt:bundle basename="org.wso2.carbon.governance.generic.ui.i18n.Resources">
-<carbon:breadcrumb
-        label="<%=request.getParameter("breadcrumb")%>"
-        topPage="false"
-        request="<%=request%>"/>
-<script type="text/javascript">
-
-/*
-     function validateAndSaveConfiguration() {
-
-        var xmlURL = editAreaLoader.getValue("payloadEditor");
-        var _schema="service-ui-config";
-
-           new Ajax.Request('../services/xmlconfig_validator_ajaxprocessor.jsp',
-            {
-                method:'post',
-                parameters: { target_xml: xmlURL,schema: _schema},
-                onSuccess: function(transport) {
-                 var returnValue = transport.responseText;
-                 if (returnValue.search(/---XMLSchemaValidated----/) != -1) {
-                   SaveConfiguration();
-                 } else {
-                     CARBON.showErrorDialog(transport.responseText);
-                 }
-                },
-
-                onFailure: function(transport) {
-                    CARBON.showErrorDialog(transport.responseText);
-                    return;
+    <carbon:breadcrumb
+            label="<%=request.getParameter("breadcrumb")%>"
+            topPage="false"
+            request="<%=request%>"/>
+    <script type="text/javascript">
+
+        /*
+         function validateAndSaveConfiguration() {
+
+         var xmlURL = editAreaLoader.getValue("payloadEditor");
+         var _schema="service-ui-config";
+
+         new Ajax.Request('../services/xmlconfig_validator_ajaxprocessor.jsp',
+         {
+         method:'post',
+         parameters: { target_xml: xmlURL,schema: _schema},
+         onSuccess: function(transport) {
+         var returnValue = transport.responseText;
+         if (returnValue.search(/---XMLSchemaValidated----/) != -1) {
+         SaveConfiguration();
+         } else {
+         CARBON.showErrorDialog(transport.responseText);
+         }
+         },
+
+         onFailure: function(transport) {
+         CARBON.showErrorDialog(transport.responseText);
+         return;
+         }
+         });
+         }
+         */
+
+        function SaveConfiguration() {
+            sessionAwareFunction(function () {
+                var CustomUIForm = document.getElementById('generic.config.form');
+                var rawconfig = editAreaLoader.getValue("payloadEditor");
+                if (rawconfig.indexOf("?>") > -1) {
+                    rawconfig = rawconfig.substring(rawconfig.indexOf("?>") + 2);
                 }
```

Comment:
They have been there since moved to github and I did the reformatting of those lines.